### PR TITLE
Doc adding scripting link

### PR DESCRIPTION
**Pull request by AurynBeorn**
- Adds link to complete guide of linked message
- Fixes markdown in links in avsitter2_scripting.md
- Reformats the script snippets

### CODE REVIEW

File: pages/avsitter2/avsitter2_scripting.md [hunk 1]
```diff
@@ -7,13 +7,13 @@ permalink: avsitter2_scripting.html
...
+AVsitter allows creators to trigger animations with their own scripts through "Link Messages" and provides the ability to add [BUTTONS](/avsitter2_avpos.html#button) to menus that send user-defined Link Messages to custom scripts. The examples below are intended only as examples, and to use them correctly you first need to have a decent knowledge of [LSL scripting](http://wiki.secondlife.com/wiki/LSL_Portal).
 
-{% include warning.html content="Don't use this section without a knowledge of <a href=\"http://wiki.secondlife.com/wiki/LSL_Portal\">LSL scripting</a>. If you do not have a knowledge of scripting then please consult your own scripter." %}
+{% include warning.html content="Don't use this section without a knowledge of [LSL scripting](http://wiki.secondlife.com/wiki/LSL_Portal\). If you do not have a knowledge of scripting then please consult your own scripter." %}
```
Comment (Sei-Lisa) [inline pages/avsitter2/avsitter2_scripting.md]: Is the `\` necessary there?

File: pages/avsitter2/avsitter2_scripting.md [hunk 2]
```diff
@@ -76,10 +76,13 @@ Also see the [SWAP notecard command](/avsitter2_avpos.html#swap).
 Sent when an avatar sits. The string field includes the SITTER # and the key field includes the avatar UUID e.g.
 
 ```js
-default{
-	link_message(integer sender, integer num, string msg, key id){
-		if(num==90060){
-			llSay(0,"Welcome, "+llKey2Name(id));
+default
+{
+	link_message(integer sender, integer num, string msg, key id)
```
Comment (Sei-Lisa) [inline pages/avsitter2/avsitter2_scripting.md]: Please change the tabs to 4 spaces too.

File: pages/avsitter2/avsitter2_scripting.md (hunk 1 above)
Comment (AurynBeorn) [reply pages/avsitter2/avsitter2_scripting.md]: It's not. I'll fix it.

File: pages/avsitter2/avsitter2_scripting.md [hunk 3]
```diff
@@ -170,4 +170,6 @@ See the following pages for AVsitter2 script examples:
 - [LSL Examples (Beginner)](avsitter2_lsl_examples_beginner.html) 
 - [LSL Examples (Advanced)](avsitter2_lsl_examples_advanced.html)
 
+{% include note.html content="Advanced scripters can [check here for the complete list of messages](https://github.com/AVsitter/AVsitter/blob/master/AVsitter2/avsitter2_link_message_reference.md). You may need to consult the AVsitter source code about what some messages do specifically." %}
```
Comment (Sei-Lisa) [inline pages/avsitter2/avsitter2_scripting.md]: It would be convenient to point out that the messages listed there may change without warning.

**Comment by Sei-Lisa** [comment]
Looks good now, thanks!

**Pull request merged**